go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/pions/rtcp
+
+go 1.12
```

Comment:
Do we really need 1.12? I don't mind, but let's not set a requirement unless we're actually using some new feature

Reply:
> The go directive in a go.mod file now indicates the version of the language used by the files within that module. It will be set to the current release (go 1.12) if no existing version is present. If the go directive for a module specifies a version newer than the toolchain in use, the go command will attempt to build the packages regardless, and will note the mismatch only if that build fails.

It automatically gets inserted by `go mod`.